SpringProject/src/main/java/com/gradproject2019/users/repository/UserRepository.java

```diff
@@ -54,4 +54,21 @@ void updateUser(@Param("id") Long id,
             "ORDER BY last_name ASC",
             nativeQuery = true)
     List<User> searchByName(@Param("query") String query);
+
+    @Query(value = "SELECT EXISTS(" +
```

Comment:
Nice work;

One small thing 
`manager_id IS NOT NULL` is not needed, MySql is not Java, this will simply result in false if your Long is null

Reply:
Ah cool, I wasn't sure after doing some research so I used that just in case. Thanks!